### ISSUE TITLE
Objects created by signals are not connect to interfaces

### ISSUE THREAD

**Issue by davidspiola** [neos/flow-development-collection#1204]
### Description

A connected signal (eg `$dispatcher->connect(Workspace::class, 'beforeNodePublishing', NodeRedirectService::class, 'createRedirectsForPublishedNode');`) object (NodeRedirectService) can not be configured via objects.yaml

### Steps to Reproduce

1. create a objects.yaml.
2. setup alternative class for signal object.
3. the configured alternative class will not be called.

#### Expected behavior

The configured alternative object should be called instead of the initial object. 

#### Actual behavior

The initial object will be call anyway

**Comment by bwaidelich** [comment]
Registration of slots happens very early in the bootstrap. I doubt that it will be possible to allow object manipulation via Objects.yaml at that state.

You might be able to unregister a previously registered slot and replace it with your one in a custom Package.php.

PS: Is `NodeRedirectService` an interface at all?

**Comment by davidspiola** [comment]
Yes, NodeRedirectServiceInterface. I thought about removing in from the slot array too, but FMPV it's not possible to unregister a already registered slot right now. There is only a Dispatcher::getSlots() method, but no way to remove one in the slots array. Do you agree?

IMO object manipulation would be much cleaner but I can’t tell if it’s even possible. The question is who is?

Either way I guess it would be very cool to have that feature.
